packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -168,3 +168,67 @@ ul {
 vaadin-notification-card vaadin-button[theme~="icon"]:not([theme~="tertiary-inline"]) {
   flex: none;
 }
+
+/* stylelint-disable order/properties-order */
+ol {
+  padding: 0;
+
+  /* stylelint-disable-next-line selector-no-qualifying-type */
+  &.sales-list {
+    $item-height: 96px;
+
+    counter-reset: li;
+    display: grid;
+    gap: var(--lumo-space-m);
+    list-style-type: none;
+
+    li {
+      border: 1px solid var(--lumo-shade-20pct);
+      border-radius: 9999px /* This is needed to create the pill shape */;
+      height: $item-height;
+      position: relative;
+
+      &::before {
+        align-items: center;
+        background-color: #000;
+        border-radius: 50%;
+        color: #fff;
+        content: counter(li);
+        counter-increment: li;
+        display: flex;
+        font-size: var(--lumo-font-size-xxxl);
+        font-weight: 800;
+        height: $item-height;
+        justify-content: center;
+        left: 0;
+        position: absolute;
+        width: $item-height;
+      }
```

Comment:
Why are we not using modern `::marker` pseudo-element styling here?

https://stackoverflow.com/a/68186109/35946

This could remove a bunch of code, so doesn't make sense to review any further for now.

Reply:
I'm not sure I see the upside to using `::marker`? I understand that it is newer, though I don't see what else it exposes. Most of the styling that is defined right now, would have to be defined in `::marker` as well.

This issue that I'm struggling with right now is how to make the typography _fluid_. I could set a breakpoint, and reduce the font size, though that may look strange on medium size devices.

Reply:
> I don't mind switching to `::marker` though I'm not sure I see the upside? I understand that it is newer, though I don't see what else it exposes. Most of the styling that is defined right now, would have to be defined in `::marker` as well.

Sure, but we should be using the best method available at any given time, avoids questions about "was there a specific reason not to" in the future.

> The issue that I'm struggling with right now is how to make the typography _fluid_. I could set a breakpoint, and reduce the font size, though that may look strange on medium size devices.

Typography fluidity is limited indeed, not the least because there really hasn't been much demand for it

There's prior art for headings https://github.com/conversionxl/aybolit/blob/master/packages/cxl-lumo-styles/scss/typography.scss

Ideally we'd avoid micro-managing these on an element level - there should be some larger object controlling typography sizes across devices. I'd say start with zero-fluid to avoid invention rabbit hole here.

Reply:
The issue I run into when setting static width/height is that it breaks on mobile. Then there is the issue of content. Too small of a pill container doesn't allow the content to fit. I've tried using `vw` as a unit, though even that doesn't scale well.